pages/contact-employment-insurance.js

```diff
@@ -229,7 +138,7 @@ export async function getStaticProps({ res, locale }) {
       breadCrumbItems,
       bannerContent: locale === 'en' ? bannerContent.en : bannerContent.fr,
       popupContent: locale === 'en' ? popupContent.en : popupContent.fr,
-      contactMethods: locale === 'en' ? contactMethods.en : contactMethods.fr,
+      pageContent: locale === 'en' ? pageContent.en : pageContent.fr,
     },
   }
 }
```

Comment:
Can also add the remaining propTypes here: breadCrumbItems, langToggleLink, etc.

Reply:
Updated.